splash/server.py

```diff
@@ -86,11 +87,12 @@ def parse_opts():
 def start_logging(opts):
     import twisted
     from twisted.python import log
-    from twisted.python.logfile import DailyLogFile
-    if opts.logfile:
-        logfile = DailyLogFile.fromFullPath(opts.logfile)
-    else:
-        logfile = sys.stderr
+    # TODO: fix this.
+    # from twisted.python.logfile import DailyLogFile
+    # if opts.logfile:
+    #     logfile = DailyLogFile.fromFullPath(opts.logfile)
+    # else:
+    logfile = sys.stderr
```

Comment:
Hmm makes sense. The twisted.python.logfile porting is stuck on some Windows only corner case.